development.json

```diff
@@ -1,57 +1,11 @@
 {
   "version": 0,
   "networks": [
-    "arbitrumrinkeby",
     "evmostestnet",
```

Comment:
ahh okay i think requirements were not made sufficiently clear; i believe we want to re-deploy dev with goerli x2. so like, goerli-1 and goerli-2 as the two networks. no rinkeby (that's being sunsetted in a year, so we don't want to focus on it) and no evmos to begin with (priority on stability and ease of use)

Reply:
Understood - I think Conner just now also told me something similar so this is now clear.